instana/instrumentation/aiohttp/server.py

```diff
@@ -8,6 +8,18 @@
 from ...util import strip_secrets
 
 
+def handle_aiohttp_exception(scope, exception, http_status_code=500):
+    logger.debug("aiohttp stan_middleware", exc_info=True)
+    if scope is not None:
+
```

Comment:
Looks like we need the 
```
if 500 <= response.status <= 511:
    scope.span.mark_as_errored()
```
part here as well, wdyt?

Reply:
its already inside the `scope.span.log_exception(exception)` apparently

Reply:
Hmm I see. But does that mean a 400/401 would get "marked as errored" as well?
```
        except aiohttp.web_exceptions.HTTPError as e:
            handle_aiohttp_exception(scope, e, e.status_code)
            raise
```
This would send any HTTP status exception (not that they're relevant for us, but including 3XXs) into this function. Maybe that `if 500 <= response.status <= 511:` line (why 511?) should be around the `scope.span.log_exception(exception)` call then.

Reply:
Side note: in my experience "being alerted for 400 when a client is broken, or for 401 when someone's making a scan" are some one of those ideas that sound good in theory, but become annoying very quickly in practice. That's why I'm trying to make sure only actual errors/failures get accounted for in the error rate (and the call analyses).